From 96d12e7753bb39062f5ce530d2daf375a4458f3f Mon Sep 17 00:00:00 2001
From: Richard Cochran <rcochran@linutronix.de>
Date: Tue, 29 Nov 2016 14:02:56 +0100
Subject: [PATCH] Treat the debian directory as precious.

In the upstream Linux build system, the debian directory is a
throwaway build product.  However, after elbe debianization, this
directory should be preserved and maintained like any other part of
the build system.  This patch removes the directory from the git
ignore file and from the clean rules.

Signed-off-by: Richard Cochran <rcochran@linutronix.de>
---
 .gitignore               | 5 -----
 scripts/package/Makefile | 3 ---
 2 files changed, 8 deletions(-)

diff --git a/.gitignore b/.gitignore
index fd3a355..d926133 100644
--- a/.gitignore
+++ b/.gitignore
@@ -52,11 +52,6 @@ Module.symvers
 /Module.markers
 
 #
-# Debian directory (make deb-pkg)
-#
-/debian/
-
-#
 # tar directory (make tar*-pkg)
 #
 /tar-install/
diff --git a/scripts/package/Makefile b/scripts/package/Makefile
index 493e226..aba11a3 100644
--- a/scripts/package/Makefile
+++ b/scripts/package/Makefile
@@ -97,9 +97,6 @@ bindeb-pkg: FORCE
 	$(MAKE) KBUILD_SRC=
 	+$(call cmd,builddeb)
 
-clean-dirs += $(objtree)/debian/
-
-
 # tarball targets
 # ---------------------------------------------------------------------------
 tar%pkg: FORCE
-- 
2.1.4

